scripts/run_integration_tests.sh

```diff
@@ -11,7 +11,6 @@ set -e
 # These values need to be in sync with serverless.yml, where there needs to be a function
```

Comment:
I would update the comment at the top of this file to document the new expected environment variable(s)